### ISSUE TITLE
run tracking and show "epoch 10" when loading model

### ISSUE THREAD

**Issue by WHAOOOOO** [ifzhang/FairMOT#82]
Why do we use the all_dla34.pth model provided to run tracking on the validation dataset from 2DMOT15 and show “epoch 10” when loading the model?

**Comment by ifzhang** [comment]
I trained the baseline model on the large training set for 30 epochs and then finetuned on MOT17 for another 10 epochs. It can get 0.2+ MOTA on MOT17 test set.
